src/main.rs

```diff
@@ -96,6 +105,41 @@ async fn main() -> std::io::Result<()> {
         .new_game(GameOptions {}, Some("test".to_string()))
         .unwrap();
 
+    let reaper_manager_ref = game_manager_wrapper.clone();
+
+    // Game Reaper thread
+    thread::spawn(move || loop {
+        thread::sleep(Duration::from_secs(REAP_INTERVAL));
+        let test_handle = GameHandle("test".to_string());
+        let num_games = reaper_manager_ref
```

Comment:
You take and release the lock just to take it again on the next line. How about when you take the lock for `game_manager_mut`, you count the games there. Also it's just a number, you shouldn't need to clone it (bc it implements Copy).